From c216c3923903c00d327302fed3c114cf5e6c095c Mon Sep 17 00:00:00 2001
From: hemx0147 <rnx781@gmail.com>
Date: Fri, 14 Oct 2022 12:23:41 +0200
Subject: [PATCH] Update Nyx host version

---
 MdePkg/Include/Library/NyxHypercalls.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdePkg/Include/Library/NyxHypercalls.h b/MdePkg/Include/Library/NyxHypercalls.h
index 41cef57da4..d1b9d6a5f2 100644
--- a/MdePkg/Include/Library/NyxHypercalls.h
+++ b/MdePkg/Include/Library/NyxHypercalls.h
@@ -109,7 +109,7 @@ static void hprintf(const char *msg){
 #define NYX_HOST_MAGIC  0x4878794e
 #define NYX_AGENT_MAGIC 0x4178794e
 
-#define NYX_HOST_VERSION 1
+#define NYX_HOST_VERSION 2
 #define NYX_AGENT_VERSION 1
 
 typedef struct host_config_s {
-- 
2.25.1

